polyfill/lib/ecmascript.mjs

```diff
@@ -1533,6 +1533,9 @@ export const ES = ObjectAssign({}, ES2019, {
   ComparisonResult: (value) => (value < 0 ? -1 : value > 0 ? 1 : value),
   GetOption: (options, property, allowedValues, fallback) => {
     if (options === null || options === undefined) return fallback;
```

Comment:
do you intend to also allow null? That seems like something that should throw too.

Reply:
I don't have an opinion about this either way. The existing "allow null" code was there before, so I assumed it was there for a good reason. If not, I can take it out.

Also, I realized that this is an abstract operation so I'll need to update the spec too.  Checking coming shortly.

Reply:
@ptomato - you added this `null` test as part of https://github.com/tc39/proposal-temporal/commit/a582e19b6d29deb97a75af8f428b53c3df7a8911. Should `null` be allowed here?

Reply:
For consistency, [GetOption](https://tc39.es/proposal-temporal/#sec-getoption) is copied from the [GetOption](https://tc39.es/ecma402/#sec-getoption) in ECMA-402 (but omitting the boolean values). I expect that when we merge the spec into ECMA-262, we'll also bring the ECMA-402 GetOption into ECMA-262. I think I actually did this in response to a comment from @ljharb (which I now can't find.)

Reply:
GetOption in 402 is invalid unless it's only ever called with an Object as `options`; all of the callsites in 402 use `ToObject` first on the object when not undefined, which throws on null.